---
 app/build.gradle.kts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index f06f31bc9..f369ffd58 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -29,13 +29,13 @@ android {
     compileSdk = 36
 
     defaultConfig {
-        applicationId = "com.github.libretube"
+        applicationId = "com.github.flextube"
         minSdk = 26
         targetSdk = 36
-        versionCode = 69
-        versionName = "31.3"
+        versionCode = 7
+        versionName = "31.3.0"
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
-        resValue("string", "app_name", "LibreTube")
+        resValue("string", "app_name", "FlexTube")
     }
 
     ksp {
@@ -72,7 +72,7 @@ android {
         getByName("debug") {
             isDebuggable = true
             applicationIdSuffix = ".debug"
-            resValue("string", "app_name", "LibreTube Debug")
+            resValue("string", "app_name", "FlexTube Debug")
         }
     }
 
-- 
2.53.0

