---
 .../github/libretube/ui/fragments/PlayerFragment.kt  | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt b/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
index 58d52c0c3..cbf916616 100644
--- a/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
+++ b/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
@@ -21,6 +21,7 @@ import android.view.SurfaceView
 import android.view.View
 import android.view.ViewGroup
 import android.view.ViewGroup.LayoutParams
+import android.widget.LinearLayout
 import androidx.activity.BackEventCompat
 import androidx.activity.OnBackPressedCallback
 import androidx.activity.result.contract.ActivityResultContracts
@@ -33,6 +34,7 @@ import androidx.core.net.toUri
 import androidx.core.os.bundleOf
 import androidx.core.os.postDelayed
 import androidx.core.view.WindowCompat
+import androidx.core.view.children
 import androidx.core.view.isGone
 import androidx.core.view.isInvisible
 import androidx.core.view.isVisible
@@ -107,6 +109,7 @@ import com.github.libretube.util.OnlineTimeFrameReceiver
 import com.github.libretube.util.PlayingQueue
 import com.github.libretube.util.TextUtils
 import com.github.libretube.util.TextUtils.toTimeInSeconds
+import com.google.android.material.button.MaterialButton
 import com.google.android.material.snackbar.Snackbar
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
@@ -409,6 +412,15 @@ class PlayerFragment : Fragment(R.layout.fragment_player), CustomPlayerCallback
         if (!PlayerHelper.showCommentSectionEnabled) {
             binding.commentsToggle.visibility = View.GONE;
         }
+        if (PlayerHelper.iconOnlyVideoBottomActionsEnabled) {
+            val optionsLayout = view.findViewById<LinearLayout>(R.id.optionsLL)
+            val maybeOptions = optionsLayout?.children ?: emptySequence()
+            val options = maybeOptions.filterIsInstance<MaterialButton>()
+            for (option in options) {
+                option.text = ""
+                option.iconPadding = 0 // fix icon centering for textless MaterialButton
+            }
+        }
         // manually apply additional padding for edge-to-edge compatibility
         activity?.getSystemInsets()?.let { systemBars ->
             with(binding.root) {
-- 
2.53.0

