---
 .../github/libretube/ui/fragments/PlayerFragment.kt  | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt b/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
index 3c4b6d1f9..9a7e78dbe 100644
--- a/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
+++ b/app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt
@@ -222,6 +222,18 @@ class PlayerFragment : Fragment(R.layout.fragment_player), CustomPlayerCallback
     private var bufferingTimeoutTask: Runnable? = null
 
     private val playerListener = object : Player.Listener {
+        override fun onRenderedFirstFrame() {
+            super.onRenderedFirstFrame()
+            val oldProgress = binding.playerMotionLayout.progress
+            val delta = if (oldProgress > 0) { -0.01f } else { 0.01f }
+            handler.post {
+                binding.playerMotionLayout.progress = oldProgress + delta
+            }
+            handler.postDelayed(150) {
+                binding.playerMotionLayout.progress = oldProgress
+            }
+        }
+
         override fun onIsPlayingChanged(isPlaying: Boolean) {
             if (PlayerHelper.pipEnabled || PictureInPictureCompat.isInPictureInPictureMode(
                     baseActivity
-- 
2.53.0

